### PR TITLE
Update FAQ

### LINKED ISSUE #181: Add actual questions and answers for FAQ page

### PR DESCRIPTION
**Pull request by subhransu**
This will fix #181

### CODE REVIEW

File: ui/src/screens/About/FAQScreen.tsx [hunk 1]
```diff
@@ -57,17 +57,83 @@ const CaretSVG = styled.svg<{ collapsed: boolean }>`
...
 const contents = [
   {
-    question: "How do I connect with a Job Provider?",
+    question: "What is JobsHippo?",
     answer:
-      "Click the Connect with button on the jobs board to launch a message screen. Then, type in your greetings and \
-    perhaps a pitch, then send the message. The recruiter will receive a notification by email about your interest.",
+      "JobsHippo is a job connection platform. We, a group of individuals, have built the service on our free time to \
+      bridge the gap between job seekers and job providers. We hope JobsHippo can help people during this time of crisis \
+      where many people have unfortunately lost their jobs.",
```
Comment (SheldonCheng) [inline ui/src/screens/About/FAQScreen.tsx]: ~where~ when

File: ui/src/screens/About/FAQScreen.tsx [hunk 2]
```diff
@@ -57,17 +57,83 @@ const CaretSVG = styled.svg<{ collapsed: boolean }>`
...
+    question: "Why am I unable to update my email address?",
+    answer:
+      "As we use LinkedIn as our authentication service, we are taking the email that is \
+      provided by LinkedIn. This ensures that the email is verified and the person has a \
+      LinkedIn profile guaranteeing the authenticity of an individual. We are not allowing \
+      users to change their emails at the moment. ",
+    collapsed: true,
+  },
+  {
+    question: "What is the site redirecting me to Auth0?",
```
Comment (SheldonCheng) [inline ui/src/screens/About/FAQScreen.tsx]: ~What~ Why

File: ui/src/screens/About/FAQScreen.tsx [hunk 3]
```diff
@@ -83,6 +149,39 @@ const contents = [
     requirement, what you are looking for, and compensation provided. No one likes a shady deal!",
     collapsed: true,
   },
+  {
+    question: "Can I mark a post a favourite?",
```
Comment (SheldonCheng) [inline ui/src/screens/About/FAQScreen.tsx]: ~a~ as favourite

File: ui/src/screens/About/FAQScreen.tsx [hunk 4]
```diff
@@ -57,17 +57,83 @@ const CaretSVG = styled.svg<{ collapsed: boolean }>`
...
+      services. By using Auth0, we have reduced our development effort at least by a month or \
+      two. In addition to reducing the development time, Auth0 also provides us with additional \
+      security features and protect us by blocking suspicious IP addresses and avoids DDoS attacks. \
+      In short, by using Auth0, your account is more secure. ",
+    collapsed: true,
+  },
+  {
+    question: "How do I connect with a Job provider?",
+    answer:
+      "Click the Connect with button on the jobs board to launch a message screen. Type in your \
```
Comment (SheldonCheng) [inline ui/src/screens/About/FAQScreen.tsx]: @subhransu Did you mean the to wrap `Connect with button` in quotes, like `'Connect with ...' button`.
Looks abit awkward now I think.

**Comment by SheldonCheng** [comment]
Thanks, looks good. Just some minor nits!

**Comment by subhransu** [comment]
@SheldonCheng addressed the comments

**Review by SheldonCheng — APPROVED** [review]

**Pull request merged**